File: neet/arrays/duplicate.cpp
#include <bits/stdc++.h>

using namespace std;

class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
		unordered_map<int, int> set;
		for (int i: nums) {
			if (set.count(i) > 0) {
				return true;
			}
			set.insert(make_pair(i, 0));
		}
		return false;
    }
};

File: neet/arrays/duplicate.py
#!/usr/bin/env python3

class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
        d = {}
        for i in nums:
            if d.get(i):
                return True
            else:
                d[i] = 1

        return False

class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
        return len(set(nums)) != len(nums)
